src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/matcher/PrisonerMatcher.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.personrecord.service.matcher
+
+import uk.gov.justice.digital.hmpps.personrecord.client.model.Prisoner
+import uk.gov.justice.digital.hmpps.personrecord.model.Person
+import uk.gov.justice.digital.hmpps.personrecord.validate.PNCIdentifier
+
+class PrisonerMatcher(val prisonerDetails: List<Prisoner>?, val person: Person) {
```

Comment:
There are some subtle variations when compared to OffenderMatcher so keep this for now, Once we have all the matchers then will work out the best way to minimise the duplication.

Reply:
👍